packages/dds/tree/src/test/feature-libraries/editable-tree/utilities.spec.ts

```diff
@@ -81,16 +55,34 @@ describe("editable-tree utilities", () => {
     });
 
     it("field utils", () => {
-        const schema = fieldSchema(FieldKinds.sequence, [stringSchema.name, int32Schema.name]);
+        const schema = arraySchema.localFields.get(EmptyKey) as FieldSchema;
```

Comment:
A safer way to read from a map and require you actually get a value is to do:
```suggestion
        const schema = arraySchema.localFields.get(EmptyKey) ?? fail("expected field");
```

This avoids the risk of accidently `as` casting to the wrong type, or silently ignoring the case where the map returned undefined, while still emaking the types work out (assuming fail returns `never`: there is such a function in our utils module, but there might also be one in our testing asserts library)

Reply:
Fixed.